### DISCUSSION TITLE
can you pls make the game for chromebook

### DISCUSSION THREAD

**Discussion by corneliuslikefnf** [FunkinCrew/Funkin#1490] [Q&A] [answered]
the lumadare version is fun on chromebook but i want to play senpai and mommy must muder

**Comment by suprstarrd** [comment, marked as answer]
Some Chromebooks have beta Linux support. You can try enabling it by following [these instructions](https://support.google.com/chromebook/answer/9145439?hl=en). From there, you can download the game from [here](https://ninja-muffin24.itch.io/funkin) and run the game from `funkin-linux-64bit.zip`.

**Comment by ghost** [comment]
An alternative is to play the HTML5 version using a HTTP tunnel or hosting it yourself.

**Comment by D00MSD4YZ** [comment]
> An alternative is to play the HTML5 version using a HTTP tunnel or hosting it yourself.

Hey, could you explain this further? Or maybe give me a link to a tutorial? Never heard of an HTTP tunnel.
